great_expectations_cloud/agent/agent.py

```diff
@@ -440,7 +440,7 @@ def _create_scheduled_job_and_set_started(self, event_context: EventContext) ->
         LOGGER.info("Creating scheduled job and setting started", extra=data)
 
         agent_sessions_url = (
-            f"{self._config.gx_cloud_base_url}/organizations/{self.get_organization_id()}"
+            f"{self._config.gx_cloud_base_url}/organizations/{self.get_organization_id(event_context)}"
```

Comment:
Does the UUID get converted to str automatically here?

Reply:
@allensallinger good question. we were able to confirm that yes it does